_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,210 @@
+
+2. After the transaction was confirmed, as part of a group of transactions contained in a newly-mined block being relayed across the network.
+
+There's no need for the node to receive the transaction a second time if it still has the first copy. Compact block relay (BIP152) can eliminate this redundancy by allowing a node to receive from its peers an ordered list of what transactions are included in a new block. With this knowledge, the node can use the transactions it has already received to partly or fully reconstruct the transactions part of the block for itself.  If the node doesn't receive all the transactions it needs to fully reconstruct the block, it requests the missing transactions from its peers, and then uses them to complete the block.
+
+![Compact Blocks diagram](https://raw.githubusercontent.com/bitcoin/bips/master/bip-0152/protocol-flow.png)
+
+Compact blocks provides three very important benefits to the network:
+
+1. By reducing the amount of bandwidth used by transaction relay nodes, compact blocks help offset the expected increase in bandwidth that will occur when the segwit capacity increases are enabled by miners.  This offset should allow nodes to continue operating on the network after segwit even if they're presently near their current bandwidth caps.
+
+2. By eliminating the bandwidth spike that occurs when nodes receive a new block, compact blocks may make it easier to keep a node operating on connections with limited peak bandwidth.  For example, several users have reported that receiving a new block slows down other important activity on their network, such as video conferencing, so some of those users shut down Bitcoin Core before starting those activities.  Compact blocks may eliminate those bandwidth spikes and make running Bitcoin Core less inconvenient for those users.
+
+3. Faster propagation of blocks across the network, by dramatically reducing the amount of data that needs to be transported when a new block is found.
+
+**More information:**
+
+- [Release notes][release notes compact blocks]
+
+- [Compact Block Relay FAQ](/en/2016/06/07/compact-blocks-faq/)
+
+- [BIP152][], which describes the compact blocks protocol
+
+- [Bitcoin FIBRE](http://bitcoinfibre.org/), an open source protocol and implementation that builds upon compact blocks to minimize latency for new block announcements between peers on managed networks.  FIBRE was designed and implemented alongside compact block relay version 1 and it is being used to test improvements for a subsequent version of compact block relay.
+
+## Fee filtering
+
+For several years now, Bitcoin Core nodes have used a minimum relay fee rate to determine what unconfirmed transactions they'll process, relay, and store in their individual memory pools.  Each node gets to decide its own minimum relay fee rate, and if they receive a transaction whose fee rate is below that limit, they don't add it to their memory pools or relay it to their other peers.
```

Comment:
I think the language in this paragraph and the next are slightly confusing.  The min relay fee that Bitcoin Core nodes have historically had isn't a true minimum, in that free transactions would still get relayed (though rate-limited) by default.

It's only due to the mempool limiting introduced in 0.12 that causes nodes to get into a state where no new transactions are accepted below some "minimum mempool fee" (which is a function of the minimum relay fee, the transactions which have been evicted from the mempool, and how much time has passed).

Specifically, it's the minimum mempool fee that is broadcast in fee filter messages.  If the mempool fee is not in effect, eg because the mempool isn't full and hasn't been for a while, then no feefilter messages would be sent, regardless of what the `-minrelaytxfee` is.